### PR TITLE
feat(limetype.interface): create PropertyOption interface and use it …

### PR DESCRIPTION
**Pull request by Befkadu1**
…in Property interface

fix Lundalogik/crm-feature#1828

### CODE REVIEW

File: packages/interfaces/src/limetype.interface.ts [hunk 1]
```diff
@@ -39,19 +39,21 @@ export interface Property {
...
     required: boolean;
     type: PropertyType;
-    options?: Array<{
-        key: string;
-        inactive: boolean;
-        text: string;
-        order?: number;
-        id?: number;
-    }>;
+    options?: PropertyOption[];
```
Comment (adrianschmidt) [inline packages/interfaces/src/limetype.interface.ts:42]: Ah! That was a simple solution! 😊👍

**Review by adrianschmidt — DISMISSED** [review]

File: packages/interfaces/src/limetype.interface.ts [hunk 2]
```diff
@@ -39,19 +39,21 @@ export interface Property {
...
     relation?: {
         getLimetype: () => Limetype;
         getBackreference: () => Property;
     };
 }
 
+export interface PropertyOption {
```
Comment (adrianschmidt) [inline packages/interfaces/src/limetype.interface.ts:49]: Last minute note from @jgroth: we should check if there's a corresponding property in the backend, and if so, name this the same as in the backend.

Comment (adrianschmidt) [reply packages/interfaces/src/limetype.interface.ts:49]: It's apparently just called `Option` inte the backend, so I think `PropertyOption` is just fine 👍

**Review by adrianschmidt — APPROVED** [review]

File: packages/interfaces/src/limetype.interface.ts (hunk 2 above)
Comment (jgroth) [reply packages/interfaces/src/limetype.interface.ts:49]: Just FYI :slightly_smiling_face: 
`Property` will be renamed to `LimeProperty` in the next major release to better align with the corresponding type in Python (https://github.com/Lundalogik/lime-web-components/blob/alpha/packages/lime-web-components/src/limetype/property.ts#L4)

The corresponding type for this is `Option` but that might be too generic here so `PropertyOption` could work, another alternative is `LimePropertyOption` :thinking:

**Pull request merged**